crossbar/worker/test/test_router.py

```diff
@@ -337,7 +338,7 @@ def test_root_not_required(self):
         def done(results):
             for item in results:
                 if not item[0]:
-                    raise item[1]
+                    return item[1]
```

Comment:
Yeah, it was a test that didn't work on Py3 (I tried to run the tests as much as possible to make sure it didn't break anything)